docassemble/ALKilnTests/data/sources/observation_steps.feature

```diff
@@ -99,23 +99,26 @@ Scenario: Test "Then I don’t continue" with an apostrophe
   And I will be told an answer is invalid
 
 @fast @o9 @table @json
-Scenario: I can match JSON page var to str
```

Comment:
I'm not sure why this scenario was removed, it should likely still be around?

Reply:
@plocket and I considered these 2 tests equivalent. The purpose of this test is to make sure ALKiln can match a variable in the json to a value that the test author expects. We concluded that it shouldn't matter how that json variable is set. The previous version of this test used a variable set by filling in a field. The current version uses a variable set by python. We couldn't think of a reason that those 2 methods are testing anything different, and we didn't want duplicate tests. Do you still think we should restore the previous test?

Reply:
Eh, yeah, I'd lean towards leaving it. I looked around, and surprisingly, there aren't any tests that seem to actually verify the value of a selection made by the user; we just say to input it and continue on. You're right that it's redundant for testing our JSON stuff, but we also need to test docassemble itself, which that test does.

Reply:
We can add it back in, but if we're going to test selections, etc, I think that test belongs in its own set of tests and not buried in a JSON Step test. Again, we can still add this back in here for the moment and just make an issue to move it later and add further tests, it just doesn't seem like this is the appropriate place for it. I'm not sure what you mean by testing docassemble itself and would love to talk more about that if we get a chance. I those tests, too, would belong in a different set of tests.

Reply:
https://github.com/SuffolkLITLab/ALKiln/issues/742